bh_modules/pascalkeywords.py

```diff
@@ -0,0 +1,14 @@
+"""
+BracketHighlighter.
+
+Copyright (c) 2013 - 2015 Isaac Muse <isaacmuse@gmail.com>
+License: MIT
+"""
+
+
+def compare(name, first, second, bfr):
+    """Differentiate 'repeat..until' and 'unit..end.' from '*..end' brackets."""
+    brackets = (bfr[first.begin:first.end], bfr[second.begin:second.end])
+    return (brackets[0] == "repeat" and brackets[1] == "until") \
+        or (brackets[0] == "unit" and brackets[1] == "end.") \
```

Comment:
Just a small remark from [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces

Hence I would add a paren around the or's instead of line continuations.